lib/config/hooks.js

```diff
@@ -13,6 +13,8 @@ module.exports = {
   'data:beforeDeleteIndex': ['write:add'],
   'data:beforeDeleteIndexes': ['write:add'],
   'data:beforeRefreshIndex': ['write:add'],
+  'data:beforeSetAutoRefresh': ['write:broadcast'],
+  'data:beforeGetAutoRefresh': ['write:add'],
```

Comment:
hmm OK, after carefully reading the code, I understand the reason of this "write:add".... but it looks very anti-intuitive. 
Maybe we should rename it in "notify" or something else ?